docs/index.html

```diff
@@ -564,5 +564,17 @@ <h2>Internationalization Considerations</h2>
         take into account when utilizing this data model.
       </p>
     </section>
+
+    <section class="appendix">
+      <h2>Acknowledgements</h2>
+      <p>
+        Portions of the work on this specification have been funded by the
+        United States Department of Homeland Security's (US DHS) Silicon Valley
+        Innovation Program under contracts 70RSAT20T00000003, and
```

Comment:
```suggestion
        Innovation Program under contracts  70RSAT20T00000003, 70RSAT20T00000031,70RSAT20T00000043, and
```

Reply:
@mkhraisha I merging @mprorock 's suggestion busted this, can you do it again?